### DISCUSSION TITLE
[Question]The web page is slow to open

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D#842] [Questions]
The web page is slow to open.  I used  ESP32S3N8R8 chip，and enabled the camera and webdav. How can I improve the speed of opening the web page.

**Comment by luc-github** [comment]
what do you mean slow to open?
what is your signal strengh?

**Comment by liqijian101** [comment]
It may be related to the signal strength. When the distance between the ESP 3D and the router is closer, the web page opening speed will be better.

**Comment by luc-github** [comment]
do you use external antenna?

**Comment by liqijian101** [comment]
Not use.
 I will buy the ESP32S3  of the external antenna next time

**Comment by luc-github** [comment]
yes external antenna is definitly better can increase 30~40% signal
